webpush_channels/tests/test_views_channel.py

```diff
@@ -157,16 +157,20 @@ def setUp(self):
         self.resp = self.app.put(self.channel_registration_url, headers=self.headers, status=202)
 
     def test_put_request_response_is_json(self):
-        self.assert_(json.loads(self.resp.body))
+        self.assertEquals(self.resp.json, {})
+        self.assertEquals(self.resp.headers['Content-Type'], 'application/json')
 
     def test_delete_request_response_is_json(self):
         resp = self.app.delete(self.channel_registration_url, headers=self.headers, status=202)
-        self.assert_(json.loads(resp.body))
+        self.assertEquals(resp.json, {})
```

Comment:
Do we want to do this just for uniformity (because we are using assert in the get request test)?